abulojoshua1/Maintenance-Tracker-Frontend
# #160690470 User should be able to modify a repair request

### What does this PR do?
- Add edit functionality to maintenance tracker requests

### Description of Task to be completed?
- Users are able to modify their maintenance requests

### How should this be manually tested?
- Log into the application
- Click on the maintenance text entry and modify it
- Refresh the page and everything should still be as you modify it
- **Note**: Requests do not take in special characters

### Any background context you want to provide?
-  I created a `EditRequestActions.js` file to edit requests.

### What are the relevant pivotal tracker stories?
- [#160690470](https://www.pivotaltracker.com/story/show/160690470)
